src/renderers/webgl/WebGLState.js

```diff
@@ -336,6 +336,10 @@ function WebGLState( gl, extensions, paramThreeToGL ) {
 
 	var maxTextures = gl.getParameter( gl.MAX_TEXTURE_IMAGE_UNITS );
 
+	var glVersion = gl.getParameter(gl.VERSION).split(' ')[1];
```

Comment:
I understand that you are simplifying the regex, do you prefer yours or a more detailed one like :

```javascript
var glVersion = parseFloat( /^WebGL\ ([0-9].[0-9])(.[0-9])?/.exec( gl.getParameter( gl.VERSION) )[ 1 ] );
var lineWidthAvailable = parseFloat( glVersion ) >= 1.0;
```

By using this one, the regex is ready to be encapsulated somewhere else when the time will come to test WebGL versions (like when WebGL 2.0 will be used enough).